src/libstore/build.cc

```diff
@@ -1457,6 +1463,20 @@ void DerivationGoal::tryToBuild()
        supported for local builds. */
     bool buildLocally = buildMode != bmNormal || parsedDrv->willBuildLocally();
 
+    auto started = [&]() {
```

Comment:
Nice! This is definitely useful

Reply:
I actually don't know where this came from!